ppmi_downloader/ppmi_downloader.py

```diff
@@ -294,7 +294,10 @@ def crawl_study_data(self, cache_file: str = "study_data_to_checkbox_id.json"):
         and their corresponding checkbox id
         """
         self.init_and_log()
-        self.html.click_button_chain(["Download", "Study Data", "ALL"])
+        self.html.click_button_chain(["Download", "Study Data"])
+        self.html.click_button(
+            '//*[@id="categoryMenuTree"]//*[@class="ida-studyData-category"][text()="ALL"]'
+        )
```

Comment:
This will click on the ALL tab button but I don't see where it will click on the ALL checkbox button. 
Also this should be included into the `Download_StudyData_ALL` function: https://github.com/LivingPark-MRI/ppmi-scraper/blob/34ff4b684fd05b22181e1a5b353704e16b410a44/ppmi_downloader/ppmi_navigator.py#L661-L674